frc42_dispatch/hasher/src/hash.rs

```diff
@@ -16,15 +14,15 @@ pub trait Hasher {
 pub struct Blake2bSyscall {}
 
 impl Hasher for Blake2bSyscall {
-    // fvm_sdk dependence can be removed using no_sdk feature
-    #[cfg(not(feature = "no_sdk"))]
+    // fvm_sdk dependence can be removed by setting default-features to false
+    #[cfg(feature = "use_sdk")]
     fn hash(&self, bytes: &[u8]) -> Vec<u8> {
         use fvm_shared::crypto::hash::SupportedHashes;
-        crypto::hash_owned(SupportedHashes::Blake2b512, bytes)
+        fvm_sdk::crypto::hash_owned(SupportedHashes::Blake2b512, bytes)
     }
 
     // stub version that avoids fvm_sdk syscalls (eg: for proc macro, or built-in actor use)
-    #[cfg(feature = "no_sdk")]
+    #[cfg(not(feature = "use_sdk"))]
```

Comment:
I understand this is not directly related to your change @lemmih but while you are here I think we can drop this `unimplemented` code block entirely if we compile Blake2bSyscall and the impl Hasher for Blake2bSyscall blocks conditionally on `use_sdk`.

Reply:
Done. Let me know what you think.